changelogs/fragments/59-mysql_replication_verify_server_cert.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- mysql_replication - add missing CHANGE MASTER TO options (https://github.com/ansible-collections/community.mysql/issues/59).
```

```suggestion
- "mysql_replication - add the new options: ``master_bind``,... (https://github.com/ansible-collections/community.mysql/issues/59)."
```
alternatively, we could use separate entries for each parameter